pandas/core/frame.py

```diff
@@ -1939,6 +1941,11 @@ def to_sparse(self, fill_value=None, kind='block'):
         >>> type(sdf)  # doctest: +SKIP
         <class 'pandas.core.sparse.frame.SparseDataFrame'>
         """
+        warning_message = """\
```

Comment:
```suggestion
        msg = "to_sparse is deprecated and will be removed in a future version"
```

Reply:
This change made `git diff upstream/master -u -- "*.py" | flake8 --diff` fail, modified it in a different way to do the job.